### DISCUSSION TITLE
What is the current project status?

### DISCUSSION THREAD

**Discussion by victorKochkarev** [slanatech/swagger-stats#207] [General]
Swagger-stats is a great project, but the last release occurred more than one year ago (Jul 02 2021). Are there any plans to support this project, at least update dependencies?

**Comment by sv2** [comment]
Yes, planning to do update shortly. Thanks for your interest!

**Comment by Luzgan** [comment]
Bumping that discussion again - I am mainly interested if there are any plans to update to prom-client ^15?
